src/dms/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/CreateDescriptorsValidation.feature

```diff
@@ -260,7 +260,7 @@ Feature: Create a Descriptor
 
         @API-014
         Scenario: 09 Post a new descriptor with an extra property (overpost)
-            Given a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
```

Comment:
Many of these scenarios previously had a `Given` without a `When` statement. We must _always_ have a `When` statement. In this case, we also have a small difference between the POST requests when performed as a `Given` compared to when performed as a `When` (look in `StepDefinition.cs` to see the difference).

Reply:
I think he was using the Given because the id extraction, but I'm seeing that both methods are assigning the id variable.